apps/sparo-lib/src/cli/commands/checkout.ts

```diff
@@ -74,7 +76,24 @@ export class CheckoutCommand implements ICommand<ICheckoutCommandOptions> {
   ): Promise<void> => {
     const { _gitService: gitService } = this;
     terminalService.terminal.writeDebugLine(`got args in checkout command: ${JSON.stringify(args)}`);
-    const { b, B, branch, startPoint } = args;
+    const { b, B, startPoint } = args;
+
+    let branch: string | undefined = args.branch;
+
+    /**
+     * Special case: sparo checkout -
```

Comment:
```suggestion
     * Special case: "sparo checkout -"
     *
     * "git checkout -" is a shortcut that checks out the previously checked out branch.
```

Worth documenting since probably very few people know about `-`